.vib/mediawiki/goss/mediawiki.yaml

```diff
@@ -0,0 +1,44 @@
+group:
```

Comment:
You could add a `command` test running `pear list` looking for the `Net_SMTP` extension to test the following command succeeded:

```
# Install required PHP extensions for SMTP support
pear install mail net_smtp
```
E.g.:
```
I have no name!@5db36b356951:/vib$ pear list
Installed packages, channel pear.php.net:
=========================================
Package          Version State
Archive_Tar      1.4.14  stable
Console_Getopt   1.4.3   stable
Mail             1.5.0   stable
Net_SMTP         1.10.1  stable
Net_Socket       1.2.2   stable
PEAR             1.10.13 stable
Structures_Graph 1.1.1   stable
XML_Util         1.4.5   stable
```